lex/gen.go

```diff
@@ -1470,3 +1473,51 @@ func (ts *TypeSchema) writeCborUnmarshalerEnum(name string, w io.Writer) error {
 
 	return nil
 }
+
+type Package struct {
+	GoPackage string
+	Prefix    string
+	Outdir    string
+}
+
+var Packages []Package = []Package{
+	Package{"bsky", "app.bsky", "api/bsky"},
+	Package{"atproto", "com.atproto", "api/atproto"},
+	Package{"chat", "chat.bsky", "api/chat"},
+	Package{"ozone", "tools.ozone", "api/ozone"},
+}
```

Comment:
in the near-ish future we are going to want to make it easy for folks to use this tooling for non-bluesky-maintained lexicons (aka, independent lexicons).

in the medium-term, it is most likely that any inter-namespace references would be from independent lexicons to one of these lexicons (aka, independent lexicons referencing `com.atproto.*` or `app.bsky.actor.profile`). so having these hard-coded in is fine.

overall I think having these hard-coded is fine for this PR, but I would guess we might want an additional command more like the previous behavior soon (coming weeks).